### DISCUSSION TITLE
bump-my-version/

### DISCUSSION THREAD

**Discussion by giscus** [callowayproject/bump-my-version#174] [Q&A] [answered]
# bump-my-version/

https://callowayproject.github.io/bump-my-version/

**Comment by ghost** [comment]
Are individual sections still required for versioning specific files? E.g:

```toml
[tool.bumpversion:file:setup.py]

[tool.bumpversion:file:README.md]

[tool.bumpversion:file:fractal_goat_foot/__init__.py]
```

**Reply by coordt** [reply, marked as answer]
Yes, but with a different format if you are using TOML:

https://callowayproject.github.io/bump-my-version/reference/configuration/#examples_2

**Reply by ghost** [reply]
Thanks!
